logstash-input-jdbc.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name = 'logstash-input-jdbc'
-  s.version         = '4.3.15'
+  s.version         = '4.3.16'
```

Comment:
IIUC this is a new feature with proper defaults to stay BWC which allows a patch release?

Reply:
Yes, it is 100% BWC.